src/cmd/get.rs

```diff
@@ -11,7 +11,7 @@ use tracing::{debug, instrument};
 #[derive(Debug)]
 pub struct Get {
     /// Name of the key to get
-    key: String,
+    pub(crate) key: String,
```

Comment:
I would prefer avoiding `pub(crate)` on struct fields, mostly because it is less idiomatic. We can add accessors.